src/mapping_igvc/CMakeLists.txt

```diff
@@ -43,12 +53,12 @@ find_package(catkin REQUIRED COMPONENTS
 ## Generate messages in the 'msg' folder
 add_message_files(
     DIRECTORY msg
-    FILES ConeObstacle.msg LineObstacle.msg Point2D.msg
```

Comment:
I believe there are other areas in the code that uses `mapping_igvc/Point2D`, one example is in `src/sb_laserscan_processing/include/ConeIdentification.h`. It's causing the build to fail on my machine, should you change those areas to use a different message type?

Reply:
Changed over everything to use `sb_geom_msgs/Point2D`